src/main/java/io/stargate/sgv2/jsonapi/service/bridge/executor/QueryExecutor.java

```diff
@@ -67,8 +67,13 @@ public Uni<QueryOuterClass.ResultSet> executeWrite(QueryOuterClass.Query query)
     QueryOuterClass.Consistency consistency = queriesConfig.consistency().writes();
     QueryOuterClass.ConsistencyValue.Builder consistencyValue =
         QueryOuterClass.ConsistencyValue.newBuilder().setValue(consistency);
+    QueryOuterClass.Consistency serialConsistency = queriesConfig.serialConsistency();
+    QueryOuterClass.ConsistencyValue.Builder serialConsistencyValue =
+        QueryOuterClass.ConsistencyValue.newBuilder().setValue(serialConsistency);
     QueryOuterClass.QueryParameters.Builder params =
-        QueryOuterClass.QueryParameters.newBuilder().setConsistency(consistencyValue);
+        QueryOuterClass.QueryParameters.newBuilder()
+            .setConsistency(consistencyValue)
+            .setSerialConsistency(serialConsistencyValue);
```

Comment:
Please extend unit test to confirm that configured serial consistency is set on writes..

Reply:
No I meant the uni test of the `QueryExecutor` class? Yes it's fine, the configuration is tested, but there is not test that configuration is being used in the `QueryExecutor`, we need this.. The main point of such tests is that if somebody tomorrow changes these lines above we would have a failing test.. Otherwise, deleting this would result in green CI, "showcasing" that is all fine.